selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java

```diff
@@ -125,35 +122,27 @@ public void createProjectTest() {
     notificationsPopupPanel.waitPopupPanelsAreClosed();
   }
 
-  @Test(priority = 1, groups = FLAKY)
+  @Test(priority = 1)
   public void deleteProjectsFromDashboardTest() {
-    switchToWindow(dashboardWindow);
+    seleniumWebDriver.switchTo().window(dashboardWindow);
     dashboard.selectWorkspacesItemOnDashboard();
     workspaces.selectWorkspaceItemName(WORKSPACE);
     workspaceDetails.selectTabInWorkspaceMenu(PROJECTS);
-    workspaceProjects.waitProjectIsPresent(WEB_JAVA_SPRING);
-    workspaceProjects.waitProjectIsPresent(CONSOLE_JAVA_SIMPLE);
-    openProjectSettings(WEB_JAVA_SPRING);
-    workspaceProjects.clickOnDeleteProject();
-    workspaceProjects.clickOnDeleteItInDialogWindow();
-    workspaceProjects.waitProjectIsNotPresent(WEB_JAVA_SPRING);
+
+    deleteProject(WEB_JAVA_SPRING);
+
     workspaceProjects.waitProjectIsPresent(SECOND_WEB_JAVA_SPRING_PROJECT_NAME);
-    openProjectSettings(CONSOLE_JAVA_SIMPLE);
-    workspaceProjects.clickOnDeleteProject();
-    workspaceProjects.clickOnDeleteItInDialogWindow();
-    workspaceProjects.waitProjectIsNotPresent(CONSOLE_JAVA_SIMPLE);
-  }
 
-  private void switchToWindow(String windowHandle) {
-    seleniumWebDriver.switchTo().window(windowHandle);
+    deleteProject(CONSOLE_JAVA_SIMPLE);
```

Comment:
shouldn't we wait on `CONSOLE_JAVA_SIMPLE` project is not present after deletion as well? `workspaceProjects.waitProjectIsNotPresent(CONSOLE_JAVA_SIMPLE);`

Reply:
Project deleting is checked at the end of ```deleteProject()``` method.

Reply:
Got it.